### PR TITLE
Multi-Reference Refinement

### PR DESCRIPTION
**Pull request by matiasgarcia91**
Set up Multi-Reference Refinement using eman2 `e2spt_classify.py` protocol
Issue: #8

### CODE REVIEW

File: eman2/protocols/__init__.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
     from protocol_tomo_extraction import EmanProtTomoExtraction
     from protocol_tomo_subtomogram_refinement import EmanProtTomoRefinement
     from protocol_tomo_initialmodel import EmanProtTomoInitialModel
+    from protocol_multi_reference_refinement import  EmanProtMultiReferenceRefinement
```
Comment (adrianq) [inline eman2/protocols/__init__.py]: @matiasgarcia91  Can you keep the 'tomo' prefix both in the file and the protocol name? 'EmanProtTomoMultiReferenceRefinement'

File: eman2/protocols/protocol_multi_reference_refinement.py [hunk 2]
```diff
@@ -0,0 +1,227 @@
+# coding=utf-8
+# **************************************************************************
+# *
+# * Authors:     Adrian Quintana (adrian@eyeseetea.com) [1]
```
Comment (adrianq) [inline eman2/protocols/protocol_multi_reference_refinement.py]: Keep just your name and mine...

File: eman2/protocols/protocol_multi_reference_refinement.py [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+from pyworkflow.protocol import STEPS_PARALLEL
+
+from eman2.convert import writeSetOfSubTomograms, getLastParticlesParams, updateSetOfSubTomograms
+import eman2
+
+from tomo.protocols import ProtTomoBase
+from tomo.objects import AverageSubTomogram, SetOfSubTomograms
+
+
+SAME_AS_PICKING = 0
```
Comment (adrianq) [inline eman2/protocols/protocol_multi_reference_refinement.py]: Is this actually used anywhere in the code?

File: eman2/protocols/protocol_multi_reference_refinement.py [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+from pyworkflow import utils as pwutils
+import pyworkflow.protocol.params as params
+import pyworkflow.em as pwem
+from pyworkflow.protocol import STEPS_PARALLEL
+
+from eman2.convert import writeSetOfSubTomograms, getLastParticlesParams, updateSetOfSubTomograms
+import eman2
+
+from tomo.protocols import ProtTomoBase
+from tomo.objects import AverageSubTomogram, SetOfSubTomograms
```
Comment (adrianq) [inline eman2/protocols/protocol_multi_reference_refinement.py]: Can you review the imports? I think we are not actually using some of them...

**Review by adrianq — COMMENTED** [review]
@matiasgarcia91 I know you are still working on this but some preliminary comments just in case...

**Comment by matiasgarcia91** [comment]
Adrian, no logre terminar de correr el protocolo, sospecho que es o un error en los parametros de entrada que no son los que espera o quizas mi maquina virtual no pueda soportar ese protocolo, alguna vez me tiro falta de memoria.

**Pull request closed without merging**